src/main/java/uk/gov/hmcts/darts/common/repository/DefenceRepository.java

```diff
@@ -2,9 +2,20 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.darts.common.entity.CourtCaseEntity;
 import uk.gov.hmcts.darts.common.entity.DefenceEntity;
 
+import java.util.Optional;
+
 @Repository
 public interface DefenceRepository extends JpaRepository<DefenceEntity, Integer> {
+    Optional<DefenceEntity> findByNameIgnoreCase(String name);
```

Comment:
can probably remove this, as discussed in another comment

Reply:
ACCEPT